actions/delegator/setup-token/dist/index.js

```diff
@@ -96,22 +96,31 @@ function run() {
                     audience: workflowRecipient,
                 },
                 github: {
-                    // TODO(#1410): Re-evaluate if we need the context.
-                    context: github,
+                    actor: process.env.GITHUB_ACTOR,
```

Comment:
I'd like to keep things in the token that we are adding to provenance (because we can verify them). If we are removing the plaintext actor and repository_owner from the provenance then we could do that.

Reply:
BTW, how does removing the plaintext `actor` help with privacy if the `actor_id` is included?